### PR TITLE
Oredictionary filter card

### PR DESCRIPTION
**Pull request by repo-alt**
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7712

### CODE REVIEW

**Comment by repo-alt** [comment]
(Needs a bit UI tinkering)

File: build.gradle [hunk 1]
```diff
@@ -45,8 +45,8 @@ buildscript {
     }
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_6
```
Comment (Glease) [inline build.gradle:48]: wait is AE2 actually Java 6 compatible? How ridiculour.

**Review by Glease — APPROVED** [review]
Nothing obviously wrong.

File: build.gradle (hunk 1 above)
Comment (mitchej123) [reply build.gradle:48]: Yup, got some weird ass crashes/errors when we upgraded it to 8

Comment (mitchej123) [reply build.gradle:48]: Hah, wonder if it'll work this time

Comment (Glease) [reply build.gradle:48]: repo distributed a few test builds on discord. Haven't heard of any crashes yet.

Comment (mitchej123) [reply build.gradle:48]: They previously worked for some people.  Trying to find the old errors

Comment (mitchej123) [reply build.gradle:48]: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/5748

https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/commit/42c60485e9b686d76360a3e619e69d2b360e4e0c

Comment (mitchej123) [reply build.gradle:48]: Make sure to test this on Thermos before merging w/ java 8 compat

File: src/main/resources/assets/appliedenergistics2/lang/en_US.lang [hunk 2]
```diff
@@ -330,7 +330,7 @@ gui.tooltips.appliedenergistics2.SchedulingModeRandom=Export items in random mod
 gui.tooltips.appliedenergistics2.ItemsStored=Items Stored: %s
 gui.tooltips.appliedenergistics2.ItemsRequestable=Items Requestable: %s
 gui.tooltips.appliedenergistics2.P2PFrequency=Frequency: %s
-gui.tooltips.appliedenergistics2.OreFilterHint=Filter using oredictionary names
+gui.tooltips.appliedenergistics2.OreFilterHint=Filter items using ore dictionary names
 gui.tooltips.appliedenergistics2.OreFilter=Oredictioary filter
```
Comment (GTNH-Afx237v7) [inline src/main/resources/assets/appliedenergistics2/lang/en_US.lang:334]: typo in line 334: needs to be `Ore Dictionary Filter`

Comment (Dream-Master) [reply src/main/resources/assets/appliedenergistics2/lang/en_US.lang:334]: https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/commit/4e9520fe1da423a02a61d0cf9562f25c6240aad0

**Comment by Tanimodori** [comment]
This card has no recipes in AE2-GTNH itself. The recipe is added by GTNH in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7933 . If you use AE2-GTNH but not with GTNH modpacks, you may lack the recipe which making this mod incomplete as a seperate mod.

**Pull request merged**